book/animations.md

```diff
@@ -1374,8 +1418,8 @@ class Browser:
 
     def composite_raster_and_draw(self):
         if not self.needs_composite and \
-            len(self.composited_updates) == 0 \
-            and not self.needs_raster and not self.needs_draw:
+            not self.needs_raster and \
+            not self.needs_draw:
```

Comment:
Why is this right / why does this matter? In what situation would we have an empty `composited_updates` (meaning, can't composite) but not need to composite? I suppose if we composited on the last frame? Is this just about speeding up idle frames? I'm confused.

Reply:
Oh, I overinterpreted. Easy PR.